### PR TITLE
Check if user is authenticated before lazy loading sequence

### PR DESCRIPTION
**Pull request by Njuelle**
## What does this PR do ?
Check if user is authenticated before lazy loading sequence, using auth `isAuthenticaded` store getter

### How should this be manually tested?

  - Step 1 : log in to an environment then log out
  - Step 2 : log in to another environment
  - Step 3 : try to switch to the first environment
  - Step 4 : check that there are no errors and no error toast displayed

### Other changes
- force Kuzzle-SDK to version 7.4.7 to avoid TS errors until the sdk patch is released

### CODE REVIEW

**Review by rolljee — APPROVED** [review]

File: src/components/Data/Layout.vue [hunk 1]
```diff
@@ -158,6 +159,10 @@ export default {
       this.dataNotFound = true
     },
     async fetchAllTheThings() {
+      if (!this.isAuthenticated) {
+        return
```
Comment (xbill82) [inline src/components/Data/Layout.vue:163]: 🤔 I feel the day the AC will fall into this case outside a login or bootstrap sequence. The page will be empty, nothing will happen, no error in the console... Just crickets.
I suggest that, for the time being, we just add a warning in the console and when the moment comes, we handle this properly.

**Review by xbill82 — APPROVED** [review]
I was wondering if we could test this case. Not sure we can.

File: src/components/Data/Layout.vue (hunk 1 above)
Comment (Njuelle) [reply src/components/Data/Layout.vue:163]: I totally agree!

Comment (Njuelle) [reply src/components/Data/Layout.vue:163]: Also, I found a way to test this case :)

**Pull request merged**